@here/harp-examples/src/getting-started_camera-animations.ts

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
There is already a cmera animation example in the three.js section. I think we can remove it.

Reply:
It works differently and is basically also a valid way of animation, but maybe a better name and place for both would do, maybe an "animations" submenu, which contains the examples?

Reply:
I moved both examples under Camera Animations